### ISSUE TITLE
1-Startsidan-Rubrik

### ISSUE THREAD

**Issue by zcode3** [chas-academy/u02-redesign-grupp-33#22]

**Comment by lemonyblossom** [comment]
Vilken bild? hur ska den se ut? Issuen saknar beskrivning.

**Comment by zcode3** [comment]
Beskrivningen står på user stories:

"
Img/p Issue - Placera bild mellan paragraf 3 & 4
justify center
"
Det är bilden under paragraferna i startsidan.

**Comment by zcode3** [comment]
Nej, bild1 är bilden på header och bild2 är bilden under paragraferna i startsidan.
